lustre_exporter_test.go

```diff
@@ -187,1124 +203,1457 @@ func blacklisted(blacklist []string, metricName string) bool {
+		{"lustre_grant_precreate_capacity_bytes", "Maximum space in bytes that clients can preallocate for objects", gauge, []labelPair{{"component", "ost"}, {"target", "lustrefs-OST0004"}}, 4.0828928e+07, false},
+		{"lustre_grant_precreate_capacity_bytes", "Maximum space in bytes that clients can preallocate for objects", gauge, []labelPair{{"component", "ost"}, {"target", "lustrefs-OST0006"}}, 4.0828928e+07, false},
+		{"lustre_soft_sync_limit", "Number of RPCs necessary before triggering a sync", gauge, []labelPair{{"component", "ost"}, {"target", "lustrefs-OST0002"}}, 16, false},
+		{"lustre_soft_sync_limit", "Number of RPCs necessary before triggering a sync", gauge, []labelPair{{"component", "ost"}, {"target", "lustrefs-OST0004"}}, 16, false},
+		{"lustre_soft_sync_limit", "Number of RPCs necessary before triggering a sync", gauge, []labelPair{{"component", "ost"}, {"target", "lustrefs-OST0006"}}, 16, false},
+		{"lustre_degraded", "Binary indicator as to whether or not the pool is degraded - 0 for not degraded, 1 for degraded", gauge, []labelPair{{"component", "ost"}, {"target", "lustrefs-OST0002"}}, 0, false},
+		{"lustre_degraded", "Binary indicator as to whether or not the pool is degraded - 0 for not degraded, 1 for degraded", gauge, []labelPair{{"component", "ost"}, {"target", "lustrefs-OST0004"}}, 0, false},
+		{"lustre_degraded", "Binary indicator as to whether or not the pool is degraded - 0 for not degraded, 1 for degraded", gauge, []labelPair{{"component", "ost"}, {"target", "lustrefs-OST0006"}}, 0, false},
+		{"lustre_blocksize_bytes", "Filesystem block size in bytes", gauge, []labelPair{{"component", "ost"}, {"target", "lustrefs-OST0002"}}, 1.048576e+06, false},
+		{"lustre_blocksize_bytes", "Filesystem block size in bytes", gauge, []labelPair{{"component", "ost"}, {"target", "lustrefs-OST0004"}}, 1.048576e+06, false},
+		{"lustre_blocksize_bytes", "Filesystem block size in bytes", gauge, []labelPair{{"component", "ost"}, {"target", "lustrefs-OST0006"}}, 1.048576e+06, false},
+		{"lustre_job_write_bytes_total", "The total number of bytes that have been written.", counter, []labelPair{{"component", "ost"}, {"jobid", "23"}, {"target", "lustrefs-OST0000"}}, 1.55500642304e+11, false},
+		{"lustre_job_write_bytes_total", "The total number of bytes that have been written.", counter, []labelPair{{"component", "ost"}, {"jobid", "24"}, {"target", "lustrefs-OST0000"}}, 2.15147593728e+11, false},
+		{"lustre_job_write_bytes_total", "The total number of bytes that have been written.", counter, []labelPair{{"component", "ost"}, {"jobid", "25"}, {"target", "lustrefs-OST0000"}}, 2.40823762944e+11, false},
+		{"lustre_job_write_bytes_total", "The total number of bytes that have been written.", counter, []labelPair{{"component", "ost"}, {"jobid", "26"}, {
```

Comment:
Teeeeechnically I think this should be in a different section (`//Health Metrics`), but I'm definitely not offended by this.

Reply:
This was tagged on the wrong line somehow... but I'm referring to line 1647 which I added as a standalone comment below. This can be disregarded!

Reply:
Ignoring like a champ